static/js/containers/TermsOfServicePage.js

```diff
@@ -31,25 +35,130 @@ class TermsOfServicePage extends React.Component {
   }
 
   render() {
-    return <div>
-      <h3>
-        MIT Micromasters Terms of Service
-      </h3>
-
-      <p>
-        Please read through the terms of service. To enroll in MIT Micromasters you
-        must agree to the terms of service.
-      </p>
-
-      <p>
-        Add terms of service here
-      </p>
-
-      <Button
-        bsStyle="success"
-        onClick={this.handleAgree.bind(this)}
-      >Agree</Button> <Button bsStyle="danger" href="/logout">Cancel</Button>
-    </div>;
+    const profile = this.props.userProfile.profile;
+    let preferredName = profile.preferredName || SETTINGS.name;
+    let linkToEdxTermsOfService = <a href="https://www.edx.org/edx-terms-service" target='_blank'>
```

Comment:
Can you inline this variable? There doesn't seem to be a reason to have this outside the rest of the HTML below